src/observability/MultiTransactionObservabilityManager.ts

```diff
@@ -0,0 +1,32 @@
+import type { TransactionObservabilityManager } from './observabilityTypes'
+
+export class MultiTransactionObservabilityManager implements TransactionObservabilityManager {
+  private readonly managers: TransactionObservabilityManager[]
+
+  constructor(managers: TransactionObservabilityManager[]) {
+    if (managers.length === 0) throw new Error('At least one manager must be provided')
```

Comment:
Please let me know if you disagree with this check, I am not really sure if it is useful to avoid having useless managers

Reply:
I don't mind it, may prevent people from forgetting to add something. if it's ever a problem, we could add forceEmpty flag later

Reply:
on a second thought, though, it may provide an easy way to turn off specific managers - e. g. locally you may be running with monitoringEnabled false. currently we handle this by forcing noop on manager side itself, but it might be easier to simply remove it from multiManager

Reply:
I see, let me remove the check then :D